compiler-rt/lib/sanitizer_common/sanitizer_libignore.h

```diff
@@ -54,20 +54,26 @@ class LibIgnore {
     char *name;
     char *real_name;  // target of symlink
     bool loaded;
+    uptr ignored_code_range_id;
   };
 
   struct LibCodeRange {
     uptr begin;
-    uptr end;
+    // A value of 0 means the associated module was unloaded.
+    atomic_uintptr_t end;
+    bool IsInRange(uptr pc) const {
+      return (pc >= begin && pc < atomic_load(&end, memory_order_relaxed));
+    }
   };
 
-  inline bool IsInRange(uptr pc, const LibCodeRange &range) const {
-    return (pc >= range.begin && pc < range.end);
-  }
+  static_assert(sizeof(LibCodeRange) == 16,
```

Comment:
This is not true of 32bit

Reply:
I don't think this assert is add much value.

Reply:
Yes I think this assert is not needed thanks!